openapi.yml

```diff
@@ -57,6 +57,26 @@ paths:
           required: false
           schema:
             type: integer
+        - name: cocina_model_json
```

Comment:
Can we just call this `cocina` or `object` or `cocina_object`?

Reply:
i went with `cocina_object` for the param name, and shortened the corresponding var name in the app and test code (but left `_json` in that name since the app code might refer to either the built model instance or the JSON that's parsed to make it).